README.md

```diff
@@ -4,114 +4,75 @@ At the end of this process, your Azure function will forward logs from an Azure
-
-Replace `<ACCOUNT-TOKEN>` with the [token](https://app.logz.io/#/dashboard/settings/general) of the account you want to ship to.
-
-Replace `<LISTENER-URL>` with your region's listener URL. If your login URL is app.logz.io, use `listener.logz.io`. If your login URL is app-eu.logz.io, use `listener-eu.logz.io`.
-
-Click **Save**.
-
-### 4. Set up data-parser.js
-
-In the _View files_ panel, click **Add** to add a new file. Name this file `data-parser.js`.
-
-Open _data-parser.js_, paste the code from [data-parser.js](src/data-parser.js) in logzio-azure-serverless, and click **Save**.
-
-### 5. _(Optional)_ Add failsafe for log shipping timeouts
-
-If the connection to Logz.io times out, you can configure logzio-azure-serverless to back up logs to Azure Blob Storage as a failsafe to prevent dropped logs.
+You can configure logzio-azure-serverless to back up logs to Azure Blob Storage.
+This prevents dropped logs if the connection to Logz.io times out.
 
 ![Function app left menu](img/function-app-menu-integrate.png)
 
 To do this, expand your function app's left menu, and then click **Integrate**.
 
-![New Blob output](img/new-output.png)
+![New Blob output](img/azure-blob-storage-outputblob.png)
 
-In the top of the triggers panel, click **New Output**, select **Azure Blob Storage**, and then click **Select**.
+In the top of the triggers panel, click **Azure Blob Storage (outputBlob)**.
 The _Azure Blob Storage output_ settings are displayed.
 
-Leave **Blob parameter name** as "outputBlob".
-Enter the **Path** and **Storage account connection** information for the Azure Blob you're sending dropped logs to, and then click **Save**.
+Leave **Blob parameter name** blank.
+Enter the blob **Path** for the Azure blob you're sending dropped logs to, and then click **Save**.
 
-**Note:** For more information on Azure Blob output binding, see [Azure Blob storage bindings for Azure Functions > Output](https://docs.microsoft.com/en-us/azure/azure-functions/functions-bindings-storage-blob#output) from Microsoft.
-
-### 6. Install logzio-nodejs
-
-In the bottom of the window, click **Console** to show the command line, then update npm to the latest version and install logzio-nodejs:
+**Important:** Make sure the blob **Path** you're using here exists or create it now.
 
-```pwsh
-npm i -g npm
-npm install logzio-nodejs
-```
-
-The logzio-nodejs installation may take a few minutes.
-You can confirm the installation is done by clicking **View files** (on the right side of the window) and finding _node_modules > logzio-nodejs_.
+**Note:** For more information on Azure Blob output binding, see [Azure Blob storage bindings for Azure Functions > Output](https://docs.microsoft.com/en-us/azure/azure-functions/functions-bindings-storage-blob#output) from Microsoft.
 
-### 7. Test your configuration
+### 4. Test your configuration
 
 In the right of the window, click **Test** to show the _Test_ panel, and then click **Run**.
```

Comment:
Maybe need to explain where is this window? The flow changed so it's not so clear

Reply:
I don't have Azure... how can the user get there?

Reply:
@imnotashrimp let's take it private